dist/index.js

```diff
@@ -2024,6 +2024,7 @@ const child_process_1 = __webpack_require__(129);
 const fs_1 = __importDefault(__webpack_require__(747));
 const DiffChecker_1 = __webpack_require__(563);
 const safeExec = (cmd) => {
+    // Remove { stdio: 'ignore' } if you want more detailed debugging
```

Comment:
this file duplicates main.tsx, so just skip to that to see the changes in a nicer formatting